nltk/tokenize/spacy_tokenizer.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+from nltk.tokenize.api import TokenizerI
+import spacy
+
+
+class SpacyTokenizer(TokenizerI):
+    """
+    The Spacy tokenizer uses spaCy library to tokenize strings.
+
+    """
+
+    def tokenize(self, s):
+        """
+        Tokenizes string using spaCy library tokenizer.
+
+        :param s: String to tokenize.
+        :type s: str
+        :return: str
+        """
+        nlp = spacy.load('en')
+        doc = nlp(s)
```

Comment:
@elfotografo007 Please consider [disabling](https://spacy.io/usage/processing-pipelines#disabling) unneeded pipeline components such as `ner`, `parser` and `tagger`. This will make a big performance difference.